hello_world.html

```diff
@@ -1,19 +1,70 @@
 <DOCTYPE HTML>
-    <html>
+<html>
+
 <h1>
     hello world!!!
-
 </h1>
-<body bgcolor='#54ff9f'>
 
-<script src="hello_world.js">
-</script>
+<head>
+<style>
+body {
+    background-color:lightblue;
+}
+
+
+            table, th, td {
```

Comment:
When you start working in our actual repos, you will find that the code formatting is very consistent. Its important to stay consistent with your code formatting for easier readability. 
For HTML, PHP, and JS files... we always use 4 space tabs.  Ross can help you update your sublime preferences so that this is done more or less by default.
Update this HTML file to follow our formatting standards, i.e. consistent 4 space tabs